### DISCUSSION TITLE
How to add: baidu, for search engine

### DISCUSSION THREAD

**Discussion by liyuxu0405** [searxng/searxng#2166] [General]
Because some content in China must be searched on Baidu!

**Comment by dalf** [comment]
There is an old PR to implement Baidu: https://github.com/searxng/searxng/pull/1530

Big issue when I've tried: after few requests, Baidu starts to return a CAPTCHA.

**Reply by rprovodenko** [reply]
how many is "a few requests"?

**Reply by dalf** [reply]
Between 5 and 10 as far as I remember

**Reply by rprovodenko** [reply]
That's within what time limit? seems quite restrictive even for regular users

**Reply by dalf** [reply]
It's because Baidu detects SearXNG as a bot.
The solution is probably complex (pick the right TLS signature, run JS, etc...).

Is there an existing project to scrap Baidu?
